pkg/images/defaults/flavor.go

```diff
@@ -121,7 +122,8 @@ func DevelopmentBuildImageFlavor() ImageFlavor {
 		ScannerDBSlimImageName: "scanner-db-slim",
 
 		ChartRepo: ChartRepo{
-			URL: "https://charts.stackrox.io",
+			URL:     "https://mirror.openshift.com/pub/rhacs/charts",
+			IconURL: "https://github.com/stackrox/stackrox/blob/master/image/templates/helm/shared/assets/Red_Hat-Hat_icon.png",
```

Comment:
1. `IconURL` like this is not going to work.
Try swap `master` with `mm/ROX-11342-helm-readme` and open the URL.
Here's how it shows up for me
![image](https://user-images.githubusercontent.com/537715/178769809-a89197fe-f1d2-4caa-9bee-55ff23f69afa.png)
You're looking for a link that would start with https://raw.githubusercontent.com/
2. Please check this icon with UI || UX folks if they would recommend different red hat.

Reply:
Great catch! I'll update the links to raw URLs.
Regarding the logo, I took that from the official [Red Hat Brandind Guide](https://www.redhat.com/en/about/brand/standards/the-hat), so I'm reasonably sure it's the right one. I can check with our branding experts to be sure, though.